data/en/features.yml

```diff
@@ -6,6 +6,14 @@
 corefeatures:
   title: "Our core features"
   columns:
+    - title: "Web-based"
+      icon: "fa fa-html5"
+      content: "You don't need extra software. The editor runs in the browser."
```

Comment:
Maybe "runs in your browser"?

(Avoiding repeating _the_)

Reply:
Makes sense